pkg/network/netconflist.go

```diff
@@ -86,13 +91,14 @@ func NewIPAMRoute(r *net.IPNet) IPAMRoute { //nolint:interfacer
 }
 
 // NewIPAMDefaultRoute creates a new IPAMDefault route of
-// 0.0.0.0/0
-func NewIPAMDefaultRoute() (IPAMRoute, error) {
-	_, n, err := net.ParseCIDR("0.0.0.0/0")
-	if err != nil {
-		return IPAMRoute{}, err
+// 0.0.0.0/0 for IPv4 or ::/0 for IPv6
+func NewIPAMDefaultRoute(isIPv6 bool) IPAMRoute {
+	_, n, _ := net.ParseCIDR(defaultIPv4Route)
+	if isIPv6 {
+		_, n, _ = net.ParseCIDR(defaultIPv6Route)
```

Comment:
I wonder if we should keep the error but panic instead of returning it - maybe even make this an `init()` to guarantee that we don't make a working Podman if we accidentally add a bad constant?

Reply:
now that you mention we can get rid of this and return a constant directly in pkg/domain/infra/abi/network.go

Reply:
will keep it as it was, with the err, to make the diff smaller, as you say it can deviate in the future but right now there is no  big gain here